modules/140-user-authz/docs/FAQ.md

```diff
@@ -295,7 +295,7 @@ If the resource belongs to a namespace, you need to extend the use role instead
  kind: ClusterRole
  metadata:
    labels:
-     rbac.deckhouse.io/aggregate-to-role: user
+     rbac.deckhouse.io/aggregate-to-kubernetes-as: user
```

Comment:
And what about FAQ_RU.md ?
https://github.com/deckhouse/deckhouse/blob/main/modules/140-user-authz/docs/FAQ_RU.md#%D1%80%D0%B0%D1%81%D1%88%D0%B8%D1%80%D0%B5%D0%BD%D0%B8%D0%B5-%D1%81%D1%83%D1%89%D0%B5%D1%81%D1%82%D0%B2%D1%83%D1%8E%D1%89%D0%B8%D1%85-use-%D1%80%D0%BE%D0%BB%D0%B5%D0%B9